addons/apecloud-mysql/templates/cr-check.yaml

```diff
@@ -0,0 +1,4 @@
+{{- $namespace := .Release.Namespace }}
+{{- if not (lookup "apps.kubeblocks.io/v1alpha1" "ComponentDefinition" $namespace .Values.etcd.etcdCrdName) }}
+  {{- fail (printf "The ComponentDefinition %s is required but does not exist in namespace %s." .Values.etcd.etcdCrdName $namespace) }}
+{{- end }}
```

Comment:
how will this check executed

Reply:
In execute helm install

Reply:
In fact, this is indeed a stopgap measure, because this design requires the etcd addon to be set up regardless of whether proxy and etcd are needed. Ideally, this determination should be made during cluster installation, but it cannot be done.

Reply:
> In fact, this is indeed a stopgap measure, because this design requires the etcd addon to be set up regardless of whether proxy and etcd are needed. Ideally, this determination should be made during cluster installation, but it cannot be done.

I see, could be left as a future work.